routers/web/home.go

```diff
@@ -59,6 +66,57 @@ func Home(ctx *context.Context) {
 	ctx.HTML(http.StatusOK, tplHome)
 }
 
+// Home renders main sitemap
+func HomeSitemap(ctx *context.Context) {
+	m := sitemap.NewSitemapIndex()
+	if !setting.Service.Explore.DisableUsersPage {
+		_, cnt, err := user_model.SearchUsers(&user_model.SearchUserOptions{
+			Actor:       ctx.User,
```

Comment:
Is it feasible to have a scenario whereby a authenticated user is going to request sitemap.xml? I'd really only expect search machines is going trough here.